.storybook/config.tsx

```diff
@@ -58,10 +59,10 @@ addDecorator(story => {
               checked={isDarkTheme}
               colored={false}
               disabled={false}
-              Label={"asdas"}
+              Label={Text}
```

Comment:
Why is label with capital?

Reply:
hmmm, i believe this is capital to indicate that this input property expects a component. Check `toggle.js `for more information.